src/geometry/euclidean/circle.lean

```diff
@@ -0,0 +1,186 @@
+/-
+Copyright (c) 2021 Manuel Candales. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Manuel Candales
+-/
+import geometry.euclidean.basic
+import geometry.euclidean.circumcenter
+import geometry.euclidean.monge_point
+import geometry.euclidean.triangle
+import linear_algebra.affine_space.midpoint
+
+/-!
+# Circles
+
+This file proves basic geometrical results about distances and angles
+in circles in real inner product spaces and Euclidean affine spaces.
+-/
+
+noncomputable theory
+open_locale big_operators
+open_locale euclidean_geometry
+open_locale real
+open_locale real_inner_product_space
+
+namespace inner_product_geometry
+/-!
+### Geometrical results on circles in real inner product spaces
+
+This section develops some results on circles in real inner product spaces,
+which are used to deduce corresponding results for Euclidean affine spaces.
+-/
+
+variables {V : Type*} [inner_product_space ℝ V]
```

Comment:
Moving this declaration of variables out of the namespace means that you won't have to redeclare them below.